lms/djangoapps/grades/models.py

```diff
@@ -25,12 +25,12 @@
 
 # Used to serialize information about a block at the time it was used in
 # grade calculation.
-BlockRecord = namedtuple('BlockRecord', ['locator', 'weight', 'max_score'])
+SerializedBlockRecord = namedtuple('SerializedBlockRecord', ['locator', 'weight', 'max_score'])
```

Comment:
As we just discussed, let's:
1. keep it as `BlockRecord`
2. update `BlockRecordSet.to_json` to serialize all the locator objects to strings before converting to json.